src/PointwiseFunctions/AnalyticSolutions/Poisson/Moustache.hpp

```diff
@@ -0,0 +1,86 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/Prefixes.hpp"  // IWYU pragma: keep
+#include "DataStructures/Tensor/Tensor.hpp"     // IWYU pragma: keep
+#include "Elliptic/Systems/Poisson/Tags.hpp"    // IWYU pragma: keep
+#include "Options/Options.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+/// \cond
+class DataVector;
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+namespace Poisson {
+namespace Solutions {
+
+/*!
+ * \brief A solution to the Poisson equation with a discontinuous first
+ * derivative.
+ *
+ * \details This implements the solution \f$u(x,y)=x\left(1-x\right)
+ * y\left(1-y\right)\left(\left(x-\frac{1}{2}\right)^2+\left(y-
+ * \frac{1}{2}\right)^2\right)^\frac{3}{2}\f$ to the Poisson equation
+ * in two dimensions, and
+ * \f$u(x)=x\left(1-x\right)\left|x-\frac{1}{2}\right|^3\f$ in one dimension.
+ * Their boundary conditions vanish on the square \f$(0,1)^2\f$ or interval
```

Comment:
Maybe I'm confused by the notation, but what is the boundary of the interval `(0, 1)`? Do you mean `[0, 1]`?